modules/virt-querying-service-for-metrics-collection.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events-monitoring/virt-exposing-custom-metrics-for-vms.adoc
+
+[id="virt-querying-service-for-metrics-collection_{context}"]
+= Querying the service for metrics collection
+
+.Procedure
+
+. Placeholder text.
+. Placeholder text.
```

Comment:
This is empty for any reason?

Reply:
This file should have been removed, it is redundant. It will be removed.

Reply:
Removed.